code.js

```diff
@@ -0,0 +1,43 @@
+//Settings
+const minSpeed = 0.5;
+const maxSpeed = 3;
+let isDragStarted = false;
+
+const speedControll = document.querySelector('.speedcontrol');
+const speedControllFilled = document.querySelector('.speedcontrol_filled');
```

Comment:
Could you separate word for classnames '.speedcontrol', '.speedvalue'.
Also, typo in word 'Controll'